### PR TITLE
[EOS-17842] Adding Logger framework in s3 mini provisioner

### PR DESCRIPTION
**Pull request by nileshgovande**
Signed-off-by: Nilesh Govande <nilesh.govande@seagate.com>

### CODE REVIEW

**Comment by gowthamchinna** [comment]
test this please

File: scripts/provisioning/s3_setup [hunk 1]
```diff
@@ -53,60 +62,93 @@ def main():
...
       PostInstallCmd(args.config).process()
-      sys.stdout.write('INFO: post_install successful.\n')
-
+      #logger.info('Post_install Command successful.\n')
     elif args.command == 'cleanup':
       from cleanupcmd import CleanupCmd
       CleanupCmd(args.config).process()
-      sys.stdout.write('INFO: cleanup successful.\n')
-      
+      #logger.info('Cleanup Command successful.\n')
```
Comment (nileshgovande) [inline scripts/provisioning/s3_setup:77]: Continue using stdout.write here and add comment

File: scripts/provisioning/s3_setup [hunk 2]
```diff
@@ -53,60 +62,94 @@ def main():
...
       PostInstallCmd(args.config).process()
-      sys.stdout.write('INFO: post_install successful.\n')
-
+      logger.info('Post_install Command successful.\n')
     elif args.command == 'cleanup':
       from cleanupcmd import CleanupCmd
       CleanupCmd(args.config).process()
-      sys.stdout.write('INFO: cleanup successful.\n')
-      
+      #Cleanup will clean the log file hence writing to stdout.
```
Comment (nileshgovande) [inline scripts/provisioning/s3_setup:77]: Add it under guard (if logger exists)

**Comment by nileshgovande** [comment]
1)Add removal of this s3deployment log file (under a flag) in cleanup phase
2)Add s3deployment log file compression in support bundle
3)Handle shell scripts output

/opt/seagate/cortx/s3/install/ldap/setup_ldap.sh
/opt/seagate/cortx/s3/install/ldap/replication/setupReplicationScript.sh
/opt/seagate/cortx/auth/scripts/create_auth_jks_password.sh
/opt/seagate/cortx/s3/scripts/s3-sanity-test.sh

**Comment by vimalraghwani20** [comment]
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/1046/
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Mini-Provisioning/job/cortx-s3server/1059/

**Pull request merged**